src/ChunkCollection.jsx

```diff
@@ -1,51 +1,77 @@
 // @ts-check
 import React, { useEffect, useState } from "react";
 
-import { api } from "./api";
 import { ChunkCollectionContext } from "./ChunkCollectionContext";
+import { api, getCachedData, storeCache } from './utils'
 
 export function ChunkCollection({
   children,
   className,
   identifier,
   limit = "",
   tags = [],
+  itemClass = "",
 }) {
-  const [[error, chunks], setResponse] = useState([undefined, []]);
+  const [chunks, setResponse] = useState([]);
+  const cacheId = identifier + limit + tags.join("")
 
   useEffect(() => {
+    // Get data from localStorage
+    let cachedChunk = getCachedData(cacheId)
+    let data = cachedChunk && JSON.parse(cachedChunk)
+    if (data) setResponse(data)
+
     const urlParams = new URLSearchParams({
       limit,
       collection_identifier: identifier,
     });
 
     tags.map((tag) => urlParams.append("tags[]", tag));
```

Comment:
I think this should be a `forEach` instead of `map`. Map is used in mutating values then stored in a new one like:

```javascript
const newValue = tags.map()
```

I think this is just iterating on the array. You can use `forEach` for this. Like:

```javascript
tags.forEach((tag) => {
  urlParams.append('tags[]', tag));
})
```